src/client/config/index.css

```diff
@@ -74,6 +74,16 @@ input {
   cursor: pointer;
 }
 
+.btnShuffle {
+  background-color: var(--button-background-color);
+  border: 0;
+  border-radius: 3px;
+  padding: 7px 10px;
+  color: var(--button-text-color);
+  margin-bottom: 5px;
+  cursor: pointer;
+}
+
```

Comment:
This whole section looks to be just a copy of the `.btn` style. Is there a reason to add this? Why not just use the existing style?

Reply:
We needed a margin in the bottom to give some space between the mobbers and the button

Reply:
In that case, you should probably use both classes and slim this css to just be what's different. Then you can just set `class="btn btnShuffle"` in the html